rmgpy/kinetics/surface.pyx

```diff
@@ -259,6 +290,11 @@ cdef class StickingCoefficientBEP(KineticsModel):
                                                                                          self.E0)
         if self.Tmin is not None: string += ', Tmin={0!r}'.format(self.Tmin)
         if self.Tmax is not None: string += ', Tmax={0!r}'.format(self.Tmax)
+        if self.coverage_dependence is not None:
+            string += f", coverage_dependence={{"
+            for species, parameters in self.coverage_dependence.items():
+                string += f"'{species}': {{'E':({parameters['E'].value}, '{parameters['E'].units}'), 'm':{parameters['m']}, 'a':{parameters['a']}}}"
```

Comment:
wouldn't this need to end with a comma, in case you had more than one?. Same in the other places this code snippet shows up.